Makefile

```diff
@@ -0,0 +1,38 @@
+SHELL := /bin/bash
+
+SUPPORTED_VERSIONS=1 6
+DEV_TARGETS = $(addprefix dev_elasticsearchv,${SUPPORTED_VERSIONS})
+
+.PHONY: requirements_es1
+requirements_es1: requirements.txt requirements-dev.txt
+	# Replace the default elasticsearch versions with the ones that work with es v1.x
+	# Create new files on disk that refer to the custom versions
+	@sed -e 's/^elasticsearch==\(.*$$\)/elasticsearch==1.9.0/g' requirements.txt | sed -e 's/^elasticsearch_dsl==\(.*$$\)/elasticsearch_dsl==0.0.11/g' > requirements.es1.txt
+	@sed -e 's/requirements\.txt/requirements.es1.txt/g' requirements-dev.txt > requirements-dev.es1.txt
+
+.PHONY: requirements_es6
+requirements_es6: requirements.txt requirements-dev.txt
+	# Since es6 is the default, no need to make any changes
+	# But make a copy so that the dev_elasticsearchv6 can be left generic
+	@cp requirements.txt requirements.es6.txt
+	@cp requirements-dev.txt requirements-dev.es6.txt
+
+.PHONY: ${DEV_TARGETS}
+${DEV_TARGETS}: dev_elasticsearchv%: requirements_es%
```

Comment:
How do I run this Makefile?  I get the following:
```
$ make
# Replace the default elasticsearch versions with the ones that work with es v1.x
# Create new files on disk that refer to the custom versions
```

I'll want to add these steps to the Readme.md

Reply:
I'll add help text, but basically if you type `make ` and hit tab, you will see all the available targets. The main thing you'll probably want to add to the readme is something to the effect of:

```
The Makefile provides targets for setting up the environment depending on your setup. Targets are provided for setting up an es1.x environment and an es6.x environment. To set up for es1.x, you would do `make dev_elasticsearchv1`, which would:
* Patch the requirements.txt to fit the requirements of your elasticsearch setup
* Create a virtual environment, installing dependencies from the generated requirements.txt
* Displays a message telling you to `source venv/bin/activate`

At this point, you're ready to run the code via `python cloudtracker.py --flags`
```